index.html

```diff
@@ -26,309 +26,340 @@
+                          </ul>
+                      </div>
+                      <button class="liston-button remove-button project-button2" type="button">See Project</button>
+                  </div>
+                  <div class="professional professional3 click-card">
+                      <div class="professional-contents">
+                          <h2 class="professional-h1">Professional Art<br>Printing Data</h2>
+                          <p class="professional-p">A daily selection of privately<br>personalized reads; no accounts
+                              or<br>sign-ups
+                              required. Has been the<br>industry's standard</p>
+                          <ul class="listons-container">
+                              <li class="listons">html</li>
+                              <li class="listons">bootstrap</li>
+                              <li class="listons">ruby</li>
+                          </ul>
+                      </div>
+                      <button class="liston-button remove-button project-button2" type="button">See Project</button>
+                  </div>
+                  <div class="professional professional1 click-card">
+                      <div class="professional-contents">
+                          <h2 class="professional-h1">Professional Art<br>Printing Data</h2>
+                          <p class="professional-p">A daily selection of privately<br>personalized reads; no accounts
+                              or<br>sign-ups
+                              required. Has been the<br>industry's standard</p>
+                          <ul class="listons-container">
+                              <li class="listons">html</li>
+                              <li class="listons">bootstrap</li>
+                              <li class="listons">ruby</li>
+                          </ul>
+                      </div>
+                      <button class="liston-button remove-button project-button2" type="button">See Project</button>
+                  </div>
+                  <div class="professional professional2 click-card">
+                      <div class="professional-contents">
+                          <h2 class="professional-h1">Professional Art<br>Printing Data</h2>
+                          <p class="professional-p">A daily selection of privately<br>personalized reads; no accounts
+                              or<br>sign-ups
+                              required. Has been the<br>industry's standard</p>
+                          <ul class="listons-container">
+                              <li class="listons">html</li>
+                              <li class="listons">bootstrap</li>
+                              <li class="listons">ruby</li>
+                          </ul>
+                      </div>
+                      <button class="liston-button remove-button project-button2" type="button">See Project</button>
+                  </div>
```

Comment:
- [ ] <b>REQUIRED:</b> The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.** Check an exampe of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.

Reply:
This issue is not properly resolved